android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -139,6 +139,7 @@
     private Dynamic textTrackValue;
     private ReadableArray textTracks;
     private boolean disableFocus;
+    private boolean preventsDisplaySleepDuringVideoPlayback;
```

Comment:
You need to set this to true, as you have specified it should be true by default in README. Good to keep it consistent with iOS.

Reply:
Is there somewhere else than https://github.com/react-native-community/react-native-video/pull/2019/files#diff-e971272a051c4ba241583904cf81ff0aR126 this should be set?

Reply:
You need to set it both for ExoPlayer and MediaPlayer, currently it's only set for MediaPlayer: https://github.com/react-native-community/react-native-video/pull/2019/files#diff-e971272a051c4ba241583904cf81ff0aR126.
So please set it to true where this commit is :)

Reply:
```suggestion
    private boolean preventsDisplaySleepDuringVideoPlayback = true;
```